app/src/main/java/org/oppia/app/player/stopexploration/StopExplorationDialogFragment.kt

```diff
@@ -32,6 +34,7 @@ class StopExplorationDialogFragment : DialogFragment() {
       .setTitle(R.string.stop_exploration_dialog_title)
       .setMessage(R.string.stop_exploration_dialog_description)
       .setPositiveButton(R.string.stop_exploration_dialog_leave_button) { _, _ ->
+        FontScaleConfigurationUtil.adjustFontSize(activity as Context, StoryTextSize.MEDIUM_TEXT_SIZE)
```

Comment:
Why is this needed? Is it to reset back to where we started? If so, could we use a lifecycle observer for this purpose, instead? I think it might be a bit more reliable if we always reset the font size whenever we exit the exploration activity rather than needing to track down all cases that can end the activity and ensure we're resetting in each place.

Reply:
Done.